backend/main/server/controllers.go

```diff
@@ -14,6 +13,52 @@ import (
 	"github.com/rs/zerolog/log"
 )
 
+type errorResponse struct {
+	statusCode int
+	modalCode  string
```

Comment:
@rvmelkonian can we update this to be `errorCode` or something similar? I think calling it `modalCode` doesn't make sense since we don't care how the front-end consumes errors. Some errors could be consumed inline for example.

Reply:
Updated.